src/pages/Landing/index.tsx

```diff
@@ -52,104 +52,124 @@ export default () => {
   };
 
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
+  const openModalMutation = async () => {
+    setIsModalOpen(true);
+    const queryClient = useQueryClient();
 
-  if (isLoading) return <Loader />;
-  return (
-    <Grid>
-      <GridItem sm={6}>
-        <Button onClick={() => setDarkmode(!darkmode)} variant='secondary'>
-          {darkmode ? 'LightMode' : 'DarkMode'}
-        </Button>
-      </GridItem>
-      <GridItem sm={6}>
-        <Button onClick={() => setIsModalOpen(true)} variant='secondary'>
-          Add New Customer
-        </Button>
-      </GridItem>
-      <Modal
-        variant={ModalVariant.small}
-        title='Add Customer'
-        isOpen={isModalOpen}
-        onClose={() => setIsModalOpen(false)}
-      >
-        <Form onSubmit={onSubmit}>
-          <Grid className={classes.inlineText}>
-            <Text>Name</Text>
-            <TextInput
-              onChange={(value) => setNewUser({ ...newUser, name: value })}
-              value={newUser.name || ''}
-              id='name'
-              type='text'
-            />
-          </Grid>
-          <Grid className={classes.inlineText}>
-            <Text>Age</Text>
-            <TextInput
-              onChange={(value) => setNewUser({ ...newUser, age: Number(value) })}
-              value={newUser.age || ''}
-              id='age'
-              type='number'
-            />
-          </Grid>
-          <Grid className={classes.inlineText}>
-            <Text>Color</Text>
-            <Select
-              onToggle={() => setSelectToggle(!selectToggle)}
-              isOpen={selectToggle}
-              onSelect={(_e, value) => {
-                if (typeof value === 'string')
-                  setNewUser({ ...newUser, color: value });
-                setSelectToggle(false);
-              }}
-              id='color'
-              variant={SelectVariant.single}
-              placeholderText='Select a color'
-              selections={newUser?.color}
-              direction={SelectDirection.up}
-            >
-              {choosableColors.map((color: string, index) => (
-                <SelectOption style={{ color }} key={index} value={color} />
-              ))}
-            </Select>
-          </Grid>
-          <Checkbox
-            label='Is this person cool?'
-            id='isCool'
-            onChange={(value) => setNewUser({ ...newUser, isCool: value })}
-            isChecked={newUser.isCool}
-          />
-          <SnazzyButton isSnazzy>
-            Submit
-          </SnazzyButton>
-        </Form>
-      </Modal>
+    const { mutate } = useMutation(openModalMutation, {
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onMutate: () => {
+
+      },
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onSuccess: () => {
+      },
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onError: () => {
+      },
+    });
```

Comment:
you could rewrite this to use a mutation, but thats a lot more than what is needed (the solution can be a lot simpler).   Currently its already using useQuery()  to fetch the data and storing it using a key of 'customers'. 

Take a look at the onSubmit() function that is defined and gets called when you submit a new customer.  What is it doing?  Does it even make sense? What should it be doing?